usr/sbin/p11sak/p11sak.c

```diff
@@ -2790,17 +2793,14 @@ static CK_RV parse_file() {
             perror("Could not find home directory of current user!");
             return CKR_CANCEL;
```

Comment:
This should not be an error.  Just fall back    to the default location.

Reply:
OK, just pushed the change ... let me know if this is what you think ...